hkmc2DiffTests/src/test/scala/hkmc2/MLsDiffMaker.scala

```diff
@@ -262,8 +263,12 @@ abstract class MLsDiffMaker extends DiffMaker:
     if showResolve.isSet then
       output(s"Resolved: ${trm.showDbg}")
     showResolvedTree.get.foreach: post =>
+      case class Unexpanded(origin: Resolvable)
+      val pre: PartialFunction[Product, Product] = 
+        case t: Resolvable if t.hasExpansion => t.instantiate
+        case t: Resolvable => Unexpanded(t.duplicate.resolve)
       output(s"Resolved tree:")
-      output(trm.showAsTree(using post))
+      output(trm.showAsTree(inTailPos = false, pre = pre)(using post))
```

Comment:
This part eases the debug process of resolution as it would explicitly mark the unresolved term `Unexpanded` in the tree.